pkg/cmd/cluster_bootstrap.go

```diff
@@ -102,8 +103,7 @@ func newClusterBootstrapCmd() *cobra.Command {
 	bootstrapCmd.Flags().StringVar(&o.kubecontext, "kubecontext", "", "The context to use from the kubeconfig (defaults to current-context)")
```

Comment:
This was used as the `context` from the Greenhouse KubeConfig. Now it is used as the context for the target cluster. The flag description should be explicit about this.

Reply:
fixed in [b5ada37](https://github.com/cloudoperators/greenhouse/pull/658/commits/b5ada375f35405832d82a8c00aae42bf7e8e30a0)